ci/build-all-steps.yml

```diff
@@ -72,3 +72,17 @@ steps:
   - powershell: ./ci/stop-collect-diagnostics.ps1
     displayName: Stop collect diagnostics
     condition: and(always(), eq(variables.collect_diagnostics, 'true'), ne(variables['numTasks'], 0))
+
+  - powershell: |
+      # Build the release branch name
+      $startdate = Get-Date -Date '2020-01-20'
+      $enddate = Get-Date
+      $difference = New-TimeSpan -Start $startdate -End $enddate
+      $sprintsSince165 = ($difference.Days/21) - (($difference.Days/21)%1) # Weirdly, PowerShell modulo is actually a remainder operator. This gets the actual modulo value.
+      $currentSprint = $sprintsSince165 + 165
+      $releaseBranch = "damccormtest/releases/m" + $currentSprint
+      # Push branch to git
+      git checkout -b $releaseBranch
+      git push https://$(GitHubPAT)@github.com/microsoft/azure-pipelines-tasks $releaseBranch
```

Comment:
I am a little confused here, why do we need to create a branch in the tasks repo?

Reply:
Its what we already do - create a release branch so its clear what got released